src/App.tsx

```diff
@@ -1,7 +1,27 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [key, setKey] = useState('');
+
+  const handleKeyPress = (event: KeyboardEvent) => {
+    setKey(event.key);
+  };
+
```

Comment:
it may be better to write a constant in useEffect as it will be re-rendered after each render